### PR TITLE
GsaGH-449 steel design task

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+namespace GsaGH.Parameters {
+  /// <summary>
+  /// <para>A Design Task is collection of specifications that guide the automated, iterative design or checking of members. A Design Task is analogous to an Analysis Task in that there can be multiple Design Tasks all of which are saved with the model. Design Tasks must be executed to carry out either a design or a check based on the parameters defined in the task. </para>
+  /// <para>In Grasshopper, it is only possible to create steel design tasks.</para>
+  /// <para>Refer to <see href="https://docs.oasys-software.com/structural/gsa/references/sbs-steeldesign/">Design Tasks</see> to read more.</para>
+  /// </summary>
+  public interface IGsaDesignTask {
+    GsaList List { get; }
+    int Id { get; }
+    string TaskName { get; }
```
Comment (tlmnrnhrdt) [inline GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs:16]: let´s rename that to just `Name`? similar to `AnalysisTask`..

File: GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GsaAPI;
+using GsaGH.Helpers;
+
+namespace GsaGH.Parameters {
+  /// <summary>
```
Comment (tlmnrnhrdt) [inline GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs:8]: do we need that comment here?

**Review by tlmnrnhrdt — CHANGES_REQUESTED** [review]

File: GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs (hunk 1 above)
Comment (kpne) [reply GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs]: done

File: GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs (hunk 2 above)
Comment (kpne) [reply GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs:8]: Yes it's being reflected in the docs generator

**Review by tlmnrnhrdt — APPROVED** [review]

**Pull request merged**